src/i18n/fr/ui.ts

```diff
@@ -35,6 +38,12 @@ export default UIDictionary({
 	// Installation Guide
 	'install.autoTab': "Automatiquement via l'ILC",
 	'install.manualTab': 'Configuration manuelle',
+	// `<DeployGuidesNav>` vocabulary
+	'deploy.sectionTitle': 'Guides de déploiement',
+	'deploy.altSectionTitle': 'Plus de guides de déploiement',
+	'deploy.filterLabel': 'Filtrer par type de déploiement',
+	'deploy.ssrTag': 'Rendu côté serveur',
```

Comment:
Do you think there's something shorter we could use here? This text is displayed as a tag on each service that supports SSR so this is quite long. Do you use SSR elsewhere in French at all?

Reply:
We technically can use SSR also here because in the sidebar the SSR guide is labeled "Rendu côté serveur (SSR)"

Reply:
Nice, because guess who didn’t take long labels into account in their styles 😬 

<img width="953" alt="image" src="https://user-images.githubusercontent.com/357379/175883514-d693c3e8-051c-453a-87bd-b94e52215d7f.png">

Reply:
```suggestion
	'deploy.ssrTag': 'SSR',
```

Reply:
OOF, i didn't saw the preview, you're totally right with this change. What a hell